sdk/iothub/azure-iot-deviceprovisioningservice/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
```

Comment:
This package being rooted in sdk/deviceprovisioningservice/ is more appropriate IMO

Reply:
sdk/deviceprovisioningservice**s** actually exists for java/js/dotnet sdk mono repos but missing in Python and I do see the conflicting takes where azure-mgmt-iothubprovisioningservices exists in sdk/iothub...